src/wasmJsMain/kotlin/io/github/oshai/kotlinlogging/ConsoleOutputAppender.kt

```diff
@@ -0,0 +1,30 @@
+package io.github.oshai.kotlinlogging
+
+public class ConsoleOutputAppender : FormattingAppender() {
+  override fun logFormattedMessage(loggingEvent: KLoggingEvent, formattedMessage: Any?) {
+    when (loggingEvent.level) {
+      Level.TRACE -> consoleLog(formattedMessage.toString())
+      Level.DEBUG -> consoleLog(formattedMessage.toString())
+      Level.INFO -> consoleInfo(formattedMessage.toString())
+      Level.WARN -> consoleWarn(formattedMessage.toString())
+      Level.ERROR -> consoleError(formattedMessage.toString())
+      Level.OFF -> Unit
+    }
+  }
+}
+
+private fun consoleLog(
+  @Suppress("UNUSED_PARAMETER") msg: String,
```

Comment:
can you explain why it's not used? Anyway I suggest to change it here to message and keep tests with msg so it's clear if it's really logging the input.

Reply:
> can you explain why it's not used?

It's used, but in a weird way (in the `js("console.log(msg)")` expression). Unfortunately, linters and IDEs sometimes don't understand this syntax, so I've suppressed potential warnings.

Reply:
I renamed `msg` to `message` and removed annotations :)